CorsixTH/Lua/hospital.lua

```diff
@@ -1181,9 +1181,11 @@ function Hospital:spawnContagiousPatient()
       self:addToEpidemic(patient)
     else
       print("Cannot create epidemic - no contagious diseases available")
+      return false
     end
   else
     print("Cannot create epidemic - no staffed reception desk")
+    return false
```

Comment:
These returns are in `Hospital:spawnContagiousPatient()` if I read the code correctly?

The function name doesn't read asif you should expect a value to be returned, also it has no `@return` documentation, so you shouldn't return a value here as far as I can see.  
If you want the function to end here (which is perhaps what you're aiming for), just `return` will do.